CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx

```diff
@@ -17,10 +21,18 @@ export default function CreateReport({ onDone }) {
   const handleCDCFileChange = (e) => {
     setCDCFile(e.target.files[0])
   }
+  const handleInputChange = (e) => {
+    setInputValue(e.target.value)
+  }
 
   const handleSubmit = async (e) => {
     e.preventDefault()
 
+    if (!inputValue) {
```

Comment:
should only check if a year is selected if the report is automatic. So after line 37, `if(isAutomatic)`.